RevenueCat/Scripts/Purchases.cs

```diff
@@ -1156,6 +1156,35 @@ public void CanMakePayments(CanMakePaymentsFunc callback)
         CanMakePayments(new BillingFeature[] { }, callback);
     }
 
+    /// <summary>
+    /// Callback function containing the result of GetAmazonLWAConsentStatus
+    /// </summary>
+    ///
+    /// <param name="hasConsented">A bool value indicating whether user has given consent to
+    /// Login with Amazon.
+    /// </param>
+    /// <param name="error">An Error object or null if successful.</param>
+    public delegate void GetAmazonLWAConsentStatusFunc(bool hasConsented, Error error);
+
+    private GetAmazonLWAConsentStatusFunc GetAmazonLWAConsentStatusCallback { get; set; }
+
+    /// <summary>
+    /// Get the Login with Amazon consent status for the current user. Used to implement one-click account creation
+    /// using Quick Subscribe.
+    ///
+    /// For more information, check the documentation:
+    /// https://developer.amazon.com/docs/in-app-purchasing/iap-quicksubscribe.html
```

Comment:
I'd recommend using the rev.cat link shortener (Rebrandly) to provide this URL, so that if Amazon moves it we can easily re-point without our SDKs falling out of sync with it

Reply:
Changed!